src/pages/MovieComments.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <div class="container">
+    <div class="cmnt-container" v-show="isAuthenticated">
```

Comment:
Sto ovde isAuthenticated ? I sto ne v-if? Kapiram da je bolje da to radis negde gde renderujes MovieComments posto ovako ti se izvrsavaju citavi hookvi komponente bespotrebno :D